server/data_common/data_adaptor.py

```diff
@@ -20,8 +20,12 @@ def __init__(self, config):
         # the following is for backwards compatability with tests
         if config is None:
             config = AppConfig()
+            assert(False)
         elif type(config) == dict:
-            config = AppConfig(**config)
+            temp = AppConfig()
```

Comment:
if the tests no longer rely on this, can we delete the backward compat code?

Reply:
Ah yes, I went to the trouble of fixing all the tests, then forgot to remove this code.   Will do.